addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
     type='text'
-    value={{value}}
-    placeholder={{placeholder}}
-    oninput={{action 'input'}}
-    onkeydown={{action 'keydown'}}
-  />
+    spellcheck=false
+    value=value
+    placeholder=placeholder
+    input=(action 'input')
+    keyDown=(action 'keydown')
+  }}
 
-  <button title='clear input' class='clear' disabled={{valueIsEmpty}} {{action 'clear'}}>&#10005;</button>
+  {{#if showReset}}
+    <i title='clear input' class='clear' {{action 'clear'}}>
```

Comment:
> I don't like embedding HTML/SVG in my JS
Me too, but making it customizable is a higher priority.  I don't see how to make the button customizable w/o exposing it as a property, given that the yield body is already used to control message the list rendering.

> I also see that you removed the ability to hide the clear button all together, in lieu of just disabling it.
Oops, didn't mean to drop that.  Your PR #3 includes this functionality, which I'd support including.